mod_form.php

```diff
@@ -138,6 +151,35 @@ public function definition() {
             $mform->addElement('static', 'additionalactivitiestwo', '', get_string('additionalactivitiestwo', 'accredible'));
         }
 
+        // Load Accredible attribute keys.
+        $attributekeys = $attributekeysclient->get_attribute_keys();
+        if (isset($attributekeys)) {
+            $attributekeyschoices = array('' => 'Select a Custom Attribute') + $attributekeys;
+            // Hidden element to check if we should disable the "gradeattributekeyname" select.
+            $mform->addElement('hidden', 'attributekysnumber', 1);
+        } else {
+            $mform->addElement('hidden', 'attributekysnumber', 0);
+        }
+
+        $mform->addElement('checkbox', 'includegradeattribute', get_string('includegradeattributedescription', 'accredible'),
+            get_string('includegradeattributecheckbox', 'accredible'));
+        if (isset( $accrediblecertificate->includegradeattribute )) {
```

Comment:
When adding a new Accredible module, the grade fields are not hidden even though the checkbox is not checked.

![Screen Shot 2022-06-03 at 14 13 39](https://user-images.githubusercontent.com/20009662/171791485-5206e0d0-b4fa-4eb1-8fc8-53d3e70da481.png)